sdks/python/apache_beam/io/external/xlang_kinesisio_it_test.py

```diff
@@ -290,7 +290,7 @@ def get_first_shard_id(self, stream_name):
       time.sleep(2)
       if i == retries - 1:
         logging.error('Could not initialize kinesis stream')
-        raise
+        raise RuntimeError()
```

Comment:
Since there's no active exception, this bare raise raises a RuntimeError. I made it explicit here.

Reply:
```suggestion
        raise RuntimeError("Unable to initialize Kinesis Stream %s. Status: %s",
            stream['StreamDescription']['StreamName'], stream['StreamDescription']['StreamStatus'])
```